### ISSUE TITLE
Generating .whl file possible?

### ISSUE THREAD

**Issue by Dipankar1997161** [pybind/cmake_example#189]
Hello,

I am new to python bindings, so i had a question regarding this library. I have all the c++ files for my project, however what i now require is a .whl file for the same so i can install the python package locally into the system.

can pybind11 do it? if so it would be great to have some insights.  Or if there is any other way to do so.

I really look forward to hearing from you.

**Comment by henryiii** [comment]
Yes. I recommend reading https://learn.scientific-python.org/development/guides/packaging-compiled/ and https://learn.scientific-python.org/development/guides/gha-wheels/ which cover this is much more detail. I'd also use github.com/pybind/scikit_build_example instead of this example as it's much simpler and easier.
